packages/@aws-cdk/aws-events-targets/test/codepipeline/integ.pipeline-event-target.expected.json

```diff
@@ -23,11 +23,11 @@
                 "kms:Disable*",
                 "kms:Get*",
                 "kms:Delete*",
+                "kms:TagResource",
```

Comment:
Is there a way to avoid this? It will cause resource update for users

Reply:
An array is ordered by definition and sadly we don't have a SET type in CFN/jsii.